File: src/Java/src/ListItemSum.java
import java.util.Arrays;

public class ListItemSum {
  public static int listItemSum(int[] list) {
    if (list.length == 1) {
      return list[0];
    }
    final int MID = list.length / 2;
    return listItemSum(Arrays.copyOfRange(list, 0, MID)) + listItemSum(Arrays.copyOfRange(list, MID, list.length));
  }
}

File: src/Python/src/list_item_sum.py
import math


def list_item_sum(list: list[int]) -> int:
    if (len(list) != 1):
        MID = math.floor(len(list) / 2)
        LEFT = list[0: MID]
        RIGHT = list[MID:]
        return list_item_sum(LEFT) + list_item_sum(RIGHT)
    return list[0]
